docs/get-started/cli-commands.mdx

```diff
@@ -499,25 +499,6 @@ Set block production time in seconds. Default: `2`
 
 ---
 
-<h4><i>restore</i></h4>
-
-<Tabs>
-  <TabItem value="syntax" label="Syntax" default>
-
-    server [--restore RESTORE]
-
-  </TabItem>
-  <TabItem value="example" label="Example">
-
-    server --restore backup.dat
-
-  </TabItem>
-</Tabs>
-
-Restore blocks from the specified archive file
-
----
-
```

Comment:
Why was this removed?

Reply:
It is a duplicate.
You can check [here](https://edge-docs.polygon.technology/docs/get-started/cli-commands) under the server commands, that we have restore twice